### DISCUSSION TITLE
GitHub scam people / dosent delete links after 2 days

### DISCUSSION THREAD

**Discussion by Maxhexa** [community/community#74989] [Other Feature Feedback, Questions, & Ideas]
I report this file 2 days ago, there is 1000 files with scam and nobody delete this. How many people must scam before delete this?

https://github.com/apps/xbox-free-gift-card-codes-list

I reported 2 days ago and support promise delete this. Wtf is wrong with you? People lose cash from credits cards

**Comment by Maxhexa** [comment]
Tell me why after 3 days many reports this links scam people. 

![IMG_5221](https://github.com/community/community/assets/129010073/df894a88-2d8f-40e5-8e72-71cfc4e8f862)
 
 
Why GitHub allowed scam people in https://github.com/apps is

I wrote 2 days ago thread in this section security and nobody delete this

**Comment by samus-aran** [comment]
Hi @Maxhexa , Thanks for joining us for the GitHub Universe celebrations in the GitHub Community, we're glad you're here! If you're looking for help with a general GitHub feature you might want to try asking in the main [GitHub Community](https://github.com/orgs/community/discussions/). We've gone ahead and re-categorised this for you. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic. Best of luck!

**Reply by SkippydaHippy79** [reply]
Not sure what to do

**Reply by SkippydaHippy79** [reply]
Help me with

**Reply by SkippydaHippy79** [reply]
~~[]()~~

**Comment by Maxhexa** [comment]
What is happens with this site? Scam is now allowed? Is 500 thread in google with scam from GitHub apps
